decision-records/007-javascript-file-formats.md

```diff
@@ -0,0 +1,84 @@
+# JavaScript file names and formats
+
+## Decision
```

Comment:
This is looking great so far. Do you think it'd be useful to include the same tree view of the structure of the package we have in https://github.com/alphagov/govuk-frontend/pull/3726 as well?